src/DotNetWorker.Core/Converters/Converter/AllowConverterFallbackAttribute.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker.Converters
+{
+    /// <summary>
+    /// An attribute that specifies if Converter fallback is allowed
+    /// </summary>
+    [AttributeUsage(
```

Comment:
Isn't this attribute only applied to binding attribute Types (i.e. classes)? Why the long list of targets?